prompt2model/utils/config.py

```diff
@@ -0,0 +1,16 @@
+"""Place to store all the default configurations."""
+MAX_SUPPORTED_BATCH_SIZE = 128
+
+# All the default hyperparameters has to be set to some standardised settings
```

Comment:
The meaning of this comment is a little unclear to me.

Also the grammar is incorrect, should be "All the default hyperparameters **have** to be set to some standardised settings." (with a period at the end)

Reply:
For now, I am deleting this comment. We can address this in the different PR.